demo/deploy-bookbuyer.sh

```diff
@@ -30,6 +30,30 @@ if [ "$DEPLOY_ON_OPENSHIFT" = true ] ; then
     fi
 fi
 
+CONTAINER_NAME="bookbuyer"
+if [ "$OS" = windows ]; then
+CONTAINER_NAME="bookbuyer-windows"
+fi
+
+echo -e "Deploy bookbuyer Service"
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  name: bookbuyer
+  namespace: $BOOKBUYER_NAMESPACE
+  labels:
+    app: bookbuyer
+spec:
+  ports:
+  - port: 14001
+    name: bookbuyer-port
+
+  selector:
+    app: bookbuyer
+EOF
+
+
```

Comment:
Per your commit message, this helps visualize the bookbuyer service. I am unsure what this means because we should ideally not be creating a service for client pods (bookbuyer is a client accessing the bookstore service), and creating this could mask bugs in the control plane.

Could you clarify why this is necessary?

Reply:
I will explain what I wanted to do and if you have any better suggestions, I am open to them.

I wanted to expose a service to the internet so we could visualize the html and display the traffic split stats. The bookbuyer seemed like a good candidate for this.

Since port forwarding is not an option for the Windows demo, I used ngnix as an ingress gateway but this can only expose services. So I made a book buyer service

Reply:
If you wanted to view the UI externally, then this seems reasonable, although be aware that the windows demo introduces a dependency on ingress, unlike the linux demo.

Could you describe in the commit message and code why the bookstore service is required because that info is missing in this PR.